packages/ui/src/components/SelectInputV2/Dropdown.tsx

```diff
@@ -136,6 +137,11 @@ const DropdownItem = styled.div<{
     outline: none;
   }
 
+
+  &:focus-visible {
+      border: ${({ theme }) => theme.colors.primary.borderWeakDisabled} 1px solid;
+    }
```

Comment:
I think the best and easiest for focus is to use the native focus of the browser instead of applying a custom one